Tests/xcprojTests/PBXProjWriterSpec.swift

```diff
@@ -49,7 +49,7 @@ class PBXProjEncoderSpec: XCTestCase {
             ">": ">".quoted,
             ";": ";".quoted,
             "&": "&".quoted,
-            "$": "$".quoted,
+            "$": "$",
```

Comment:
Could you add the example text from #205

Reply:
Done